File: C11/백준/Bronze/2577. 숫자의 개수/숫자의 개수.c
#include <stdio.h>

int main(void){
	
	int A, B, C;
	scanf("%d %d %d", &A, &B, &C);
	
	int result = A * B * C;
	int counts[10] = {0};
	
	while (result != 0){
		int digit = result % 10;
		counts[digit]++;
		result = result / 10;
	}
	
	for (int i = 0; i < 10; i++){
		printf("%d\n", counts[i]);
	}
	
	return 0;
}

File: Java/백준/Bronze/2577. 숫자의 개수/숫자의 개수.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {

        Scanner s = new Scanner(System.in);

        int A = s.nextInt();
        int B = s.nextInt();
        int C = s.nextInt();

        int result = A * B * C;
        int[] counts = new int[10];

        while (result != 0) {
            int digit = result % 10;
            counts[digit]++;
            result = result / 10;
        }

        for (int i = 0; i < 10; i++) {
            System.out.println(counts[i]);
        }

        s.close();
    }
}